cmd/minikube/cmd/delete_test.go

```diff
@@ -154,6 +158,17 @@ func TestDeleteProfile(t *testing.T) {
 	}
 }
 
+var DeleteHostAndDirectoriesMock = func(api libmachine.API, cc *config.ClusterConfig, profileName string) error {
```

Comment:
this needs to be private as well and rename to the same name as the other with "hostAndDirsDeleterMock"